### PR TITLE
Add metrics cert redeploy process

### PR DESCRIPTION
**Pull request by lpsantil**

### CODE REVIEW

**Comment by lpsantil** [comment]
@sal2fcVir

**Comment by ahardin-rh** [comment]
@lpsantil Thanks! Is this tied to a specific development card or BZ (just to help with tracking)? Also, is this targeting OCP 3.5 docs, or future 3.6 docs?

File: admin_solutions/index.adoc [hunk 1]
```diff
@@ -17,5 +17,6 @@ In this initial version of the guide, you can learn how to:
 - xref:../admin_solutions/user_role_mgmt.adoc#share-templates-cluster[Share templates across the cluster]
 - xref:../admin_solutions/user_role_mgmt.adoc#create-cluster-admin[Create a cluster administrator account]
 - xref:../admin_solutions/authentication.adoc#admin-solutions-authentication[Configure authentication providers]
+- xref:../admin_solutions/certificate_management.adoc[Certificate Management]
```
Comment (ahardin-rh) [inline admin_solutions/index.adoc]: xref:../admin_solutions/certificate_management.adoc#admin-solutions-certificate-management[Manage certificates]

**Comment by lpsantil** [comment]
This was something that came up in openshift-sme mailing list [0].  It was something that I had laying around in an EJ.  @christianh814 verified that it worked and @sferich888 asked that it get added to the docs [1].  I have no trello card or BZ for it.  It's OOTB functionality AFAICT & should work that way for a long time to come.

Ostensibly, I see no reason why the example with metrics wouldn't work in everything in 3.3+.  Might even work in 3.2.  As for 3.6, I don't know.  Metrics and Logging keep changing.  But, it's only an example.  Remove the references to Metrics and it's completely generic for apps that are using self-signed certs but want to have clients only see a CA trusted cert after the initial deployment.  For now, metrics & logging are a good examples because it would be common desire to fix the self-signed certs in Post-Install rather than complicate the Initial Install.

[0] http://post-office.corp.redhat.com/archives/openshift-sme/2017-June/msg00003.html

[1] http://post-office.corp.redhat.com/archives/openshift-sme/2017-June/msg00009.html

File: admin_solutions/index.adoc (hunk 1 above)
Comment (lpsantil) [inline admin_solutions/index.adoc]: You want me to change that link?

Comment (ahardin-rh) [inline admin_solutions/index.adoc]: @lpsantil Yes, please. We need to include the anchor ID in the path so that the docs build correctly. Thanks!

Comment (ahardin-rh) [inline admin_solutions/index.adoc]: @lpsantil Now that I think of it, there are a few minor things that I need to do with this PR, but I can take care of those things for you by fetching your branch. I'll update the link as part of my process. Thanks!

**Comment by ahardin-rh** [comment]
@lpsantil Thank you! I will apply this to 3.3, 3.4, and 3.5. As mentioned, I will fetch your branch to make a few minor edits and get this moving forward.

**Comment by ahardin-rh** [comment]
Continuing work in https://github.com/openshift/openshift-docs/pull/4552

**Pull request closed without merging**